docs(ServiceAccount): Extend example and add paragraph on match limitations (#2290)
grafana/grafana-operator: examples/serviceaccounts/_index.md

weight: 80
---

`GrafanaServiceAccounts`(SA) are quite unique compared to other resources to the security implications that can arise.
In order to avoid roque accounts in Grafana instances, the creation and matching of `SA` is at the time of writing intentionally limited.

Any `SA` matches exactly one Grafana instance through the `.spec.instanceName` field.
The `instanceName` being equal `.metadata.name`.

> Additionally, the matching is limited within the same namespace as shown below.

Reviewer: ```suggestion
Additionally, service accounts are only supported in the same namespace as shown below.
```
